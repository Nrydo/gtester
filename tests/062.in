2862410
390056321
382034791