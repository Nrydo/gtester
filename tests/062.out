3071748476676836